x/metadata/client/cli/cli_test.go

```diff
@@ -3919,11 +3880,11 @@ func (s *IntegrationCLITestSuite) TestGetCmdAddNetAssetValues() {
 		{
 			name:   "invalid net asset string",
 			args:   argsWStdFlags(scopeID, "invalid"),
-			expErr: "invalid net asset value, expected coin,volume",
+			expErr: "invalid coin invalid",
```

Comment:
This test is a little bit hard to decipher since the supplied name is actually in the error. Can we adjust either the error message or the denom?

Reply:
The comment below should help resolve this if you still want to use the name "invalid".